uint8 id
#目标的id及目标的各项参数：长宽高、目标角度
geometry_msgs/PoseWithCovariance position
geometry_msgs/TwistWithCovariance relative_velocity
geometry_msgs/AccelWithCovariance relative_acceleration

float32 length
float32 width
float32 orientation_angle

float32 rcs
int8 dynamic_property
int8 class_type
int8 meas_state
int8 prob_of_exist